modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -198,110 +198,6 @@ examplecluster-compute-1          Running                          165m    opens
 ----
 <1> This is the control plane machine for the unhealthy node, `examplecluster-control-plane-2`.
 
-.. Save the machine configuration to a file on your file system:
```

Comment:
Probably not worth holding up this PR, but after removing this substep, the [previous substep](https://github.com/openshift/openshift-docs/pull/80267/files#diff-33208a1472920fee85adcb90269b5d140d0d2c19752c3ed7bc2412df2774e047R180) should become a unnumbered step (i.e. with an asterisk) as it is the only substep. 
 
.. Obtain the machine for the unhealthy member.

Reply:
Actually, the wording in [step 4](https://github.com/openshift/openshift-docs/pull/80267/files#diff-33208a1472920fee85adcb90269b5d140d0d2c19752c3ed7bc2412df2774e047R176-R178) doesn't make sense after you remove the substeps, as all you are doing is listing nodes to identify the unhealthy member.

Reply:
@kowen-rh PTAL ^^^^

Reply:
Nice catch, thank you!

Reply:
@mburke5678 I ended up reorganizing this slightly so that the previous substep is now its own step. This makes sense to me and reads well IMO.